src/pages/ComponentsPage/ComponentsPage.sass

```diff
@@ -1,3 +1,12 @@
 @import '../../styles/core'
 
 .ComponentsPage
+    .control-bar
```

Comment:
Just curious why you need these styles? I don't see `control-bar` anywhere else in this PR..